openhcl/build_info/src/lib.rs

```diff
@@ -35,6 +39,16 @@ impl BuildInfo {
             } else {
                 ""
             },
+            internal_scm_revision: if let Some(r) = option_env!("INTERNAL_GIT_SHA") {
```

Comment:
I'm not sure I love the "internal" moniker, maybe "parent" would be better? I'm curious what others think.